apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/AnyAssertionSamples.kt

```diff
@@ -50,4 +61,188 @@ class AnyAssertionSamples {
             }
         }
     }
+
+    @Test
+    fun notToBe() {
+        expect(2) notToBe 3
+
+        fails {
+            expect(12) notToBe 12
+        }
+
+        expect(listOf(2)) notToBe listOf(3)
+
+        fails {
+            expect(listOf(2)) notToBe listOf(2)
+        }
+    }
+
+    @Test
+    fun isSameAs() {
+        val list = listOf(3)
+        expect(list) isSameAs list
+
+        fails {
+            expect(listOf(3)) isSameAs listOf(3) // fails because isSameAs is based on identity, use toBe for equality
+        }
+    }
+
+    @Test
+    fun isNotSameAs() {
+        expect(listOf(2)) isNotSameAs listOf(2) // holds because isSameAs is based on identity, use notToBe for equality
+
+        fails {
+            val list = listOf(3)
+            expect(list) isNotSameAs list // fails because isNotSameAs is based on identity, use toBe for equality
+        }
+    }
+
+    @Test
+    fun toBeNullIfNullGivenElse() {
+        expect<Int?>(null) toBeNullIfNullGivenElse null
+
+        expect<Int?>(1) toBeNullIfNullGivenElse { // subject inside this block is of type Int (actually 1)
+            it isLessThan 2
+        } // subject here is remains to type Int
+
+        fails {
+            expect<Int?>(1) toBeNullIfNullGivenElse { // subject inside this block is of type Int (actually 1)
+                it isLessThan 0
+            } // subject here is remains to type Int
+        }
+    }
+
+    @Test
+    fun notToBeNullFeature() {
+        expect<Int?>(1) notToBeNull o isLessThan 2
+        //                           | subject is now of type Int (actually 1)
+
+        fails {
+            expect<Int?>(null) notToBeNull o isLessThan 2
+            //                              |             | not reported because `notToBeNull` already fails
+            //                              | fails
+        }
+    }
+
+    @Test
+    fun notToBeNull() {
+        expect<Int?>(1) notToBeNull { // subject is now of type Int
+            it isGreaterThan 0
+            it isLessThan 10
+        } toBe 1 // subject here is remains to type Int
```

Comment:
here the text is correct and IMO better than what we currently have in api-fluent :+1: 
please fix the typo though:
```suggestion
        } toBe 1 // subject here remains type Int
```
same same in other places